### ISSUE TITLE
Simple set up issues

### ISSUE THREAD

**Issue by willdebolla** [jamiehaywood/hl#11]
Have just stumbled upon this repo and looks great - however I don't know much about Node/JS and am struggling to get it working. I have done a bit of Googling and fiddling but haven't got anywhere, so I'm going back to basics and what I'm sure is a really simple question... would appreciate someone's help!

Steps:
- Take a clean clone of the repo
- Put the code from the 'usage' section of the readme into a new file called index.js, at the root of the repo
- run `node index.js` from the root of the repo

I'm using Node v16.14.0 and get the following:

```
(node:18760) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
C:\Users\********\Documents\GitHub\hargreaves\index.js:1
import HL from 'hargreaves-lansdown'
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

Had a look in package.json re the suggestion of "type": "module", but there's already "types" in there (plus just adding this doesn't fix it), so I'm not convinced with my basic level of knowledge.

I'm sure this is really simple to solve - would be great to have instructions to allow even the newest person to get up and running.

**Comment by jamiehaywood** [comment]
Hi @willdebolla, thanks for checking out the repo!

My apologies, I had made one of the dependencies use the `latest` version (rookie error...). 

I have fixed this and updated the package to `0.0.5` to resolve this issue. You should now be able to copy the snippet in the readme, using `"type": "module",` inside your `package.json`.

Let me know if you have any more issues. If it's all sorted for you, please close this issue 🙂.

**Comment by willdebolla** [comment]
Hi @jamiehaywood thanks for looking into this so quickly. I have it up and running now 👍 .
